### PR TITLE
enable lz4

### PR DESCRIPTION
**Pull request by muelli**
I think it fails. Let's see what the bots say.

### CODE REVIEW

File: org.wireshark.Wireshark.yml [hunk 1]
```diff
@@ -14,6 +14,111 @@ finish-args:
...
+        # I don't know whether we need this and if so, where to put those files
+        #- type: file
+        #  url: https://nuget.org/api/v2/package/MaxMind.Db/2.0.0
+        #  dest-filename: thirdparty/download/MaxMind.Db.zip
+        #  sha256: 98595abd21c735fe8bb84911a7b254f5d37c910818411f8edded7378e3e7c181
+
+        #- type: file
+        #  url: https://geolite.maxmind.com/download/geoip/database/GeoLite2-Country_20181211.tar.gz
+        #  dest: thirdparty/download/
+        #  sha256: 9e71cdb9cd870e1209001cbcfb0cb4dd3a58ff2d8ab85320682680aab0c250fc
```
Comment (Lekensteyn) [inline org.wireshark.Wireshark.yml]: If the code is compiled in, the user can load the databases themselves. For out-of-the-box support, you'll have to install extra files. Using `tshark -G folders`, you can learn the search path:
```
...
MaxMind database path:  /usr/share/GeoIP
MaxMind database path:  /var/lib/GeoIP
```
Example of placement:
https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/geoip2-database

Comment (Lekensteyn) [reply org.wireshark.Wireshark.yml]: It looks like those paths are hardcoded (see `epan/maxmind_db.c`). For now you will have to patch the source I think.

File: org.wireshark.Wireshark.yml [hunk 2]
```diff
@@ -14,6 +14,111 @@ finish-args:
...
+  - name: snappy
+    cleanup:
+        - /share/doc
+        - /include
+    sources:
+        - type: archive
+          url: https://github.com/google/snappy/releases/download/1.1.3/snappy-1.1.3.tar.gz
+          sha256: 2f1e82adf0868c9e26a5a7a3115111b6da7e432ddbac268a7ca2fae2a247eef3
+
+  - name: l4z
```
Comment (Lekensteyn) [inline org.wireshark.Wireshark.yml]: Shouldn't this be "lz4" insteade of "l4z"?

Comment (muelli) [reply org.wireshark.Wireshark.yml]: good call.

File: org.wireshark.Wireshark.yml (hunk 1 above)
Comment (muelli) [reply org.wireshark.Wireshark.yml]: FTR: I've opened https://github.com/flathub/org.wireshark.Wireshark/issues/17 to track GeoIP support

**Pull request closed without merging**